data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -835,5 +874,15 @@ def multiply_HARMONIZED_GROSS_SPEND_by_thousand(
         in those countries were could be necessary due in some countries like APAC countries
         usually gross values are without multiply by thousand.
         """
-        df[comp_harm_constants.GROSS_SPEND_COLUMN] = df[comp_harm_constants.GROSS_SPEND_COLUMN] * 1000
-        return df
+        return self.multiply_values_in_given_column_by_thousand(df, comp_harm_constants.GROSS_SPEND_COLUMN)
+
+    def round_up_HARMONIZED_GROSS_SPEND_column_to_two_decimals(
```

Comment:
Let's rename this to "trim_HARMONIZED_GROSS_SPEND_column_to_two_decimals(...)"

Reply:
Renamed.

Reply:
Done